src/Create/Create.tsx

```diff
@@ -1,11 +1,172 @@
-import React, { Component } from "react";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+import { JournalEntry } from "../journal";
+import { useJournal } from "../JournalContext";
+
+function JournalEntryForm(){
+
+    const { addJournalEntry } = useJournal();
+
+    const [journalEntry, setJournalEntry] = useState<JournalEntry>({
+      date: "",
+      title: "",
+      hoursActive: 0,
+      hoursSleeping: 0,
+      hoursFocused: 0,
+      hoursOnScreen: 0,
+      hoursOutside: 0,
+      hoursReading: 0,
+      mood: 5,
+      created: new Date().toISOString(),
+      reflection: "",
+      updated: null
+    });
+    
+    // for navigation back to home page after submitting
+    const navigate = useNavigate();
+
+    const handleInputChange = (event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+        const { name, value } = event.target;
+    
+        // update journal on change
+        setJournalEntry({
+          ...journalEntry,
+            [name]: (name.startsWith("hours") || name === "mood") // for hours counts and mood, get value and handle empty string
+                ? (value === "" ? "" : Number(value))
+                : value,
+            updated: new Date().toISOString(), // for update I will need to change this. Currently can only create entries
+            created: new Date().toISOString()
```

Comment:
these fields could be moved into the `handleSubmit` callback since they only need to be updated once when we actually add the entry to the journal.